src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/CachedBitmap.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using Gdip = System.Drawing.SafeNativeMethods.Gdip;
+
+namespace System.Drawing.Imaging
+{
+    /// <summary>
+    /// Stores a <see cref="Bitmap"/> in a format that is optimized for display on a particular device.
+    /// </summary>
+    public sealed partial class CachedBitmap : IDisposable
+    {
+        internal static bool IsSupported = IsCachedBitmapSupported();
+        internal IntPtr nativeCachedBitmap;
```

Comment:
Make this a property?

Reply:
They seem to do [both](https://source.dot.net/#q=System.Drawing.Common%20native) in System.Drawing. I'm not sure what the benefits of using a property here are.

Reply:
Ah, ok. Then stay consistent within this package and leave it as is.

Otherwise it's the usual convention in the libraries to use fields only for private / protected members.